app/javascript/wikiapi.js

```diff
@@ -535,6 +541,18 @@ var wikidata = function(spec) {
         'type': 'statement',
         'rank': 'normal',
       },
+      {
+        'mainsnak': {
+          'snaktype': 'value',
+          'property': that.getPropertyID('instance of'),
+          'datavalue': {
+            'value': getItemValue(that.getItemID('human')),
+            'type': 'wikibase-entityid'
+          }
+        },
+        'type': 'statement',
+        'rank': 'normal',
+      },
```

Comment:
I don't think it's required for this PR, but just to note that we should think about factoring out a helper to generate these JSON structures representing claims. There's quite a lot of refactoring that could be done to improve this file, though, so it might be worth tackling all of those in one effort, along with writing tests for it.